Gems/AtomLyIntegration/CommonFeatures/Code/Include/AtomLyIntegration/CommonFeatures/Material/EditorMaterialSystemComponentNotificationBus.h

```diff
@@ -24,6 +24,7 @@ namespace AZ
         public:
             static const AZ::EBusAddressPolicy AddressPolicy = AZ::EBusAddressPolicy::Single;
             static const AZ::EBusHandlerPolicy HandlerPolicy = AZ::EBusHandlerPolicy::Multiple;
+            using MutexType = AZStd::recursive_mutex;
```

Comment:
missing include for recursive mutex

Reply:
Added the include and changed to regular mutex.